src/ui/starter-select-ui-handler.ts

```diff
@@ -893,6 +905,18 @@ export default class StarterSelectUiHandler extends MessageUiHandler {
       && !(starterData.passiveAttr & PassiveAttr.UNLOCKED);
   }
 
+  /**
+   * Determines if a passive upgrade is maxed for the given species ID
+   * @param speciesId The ID of the species to check the passive of
+   * @returns true if the passive has been maxed
+   */
+  isValueReductionMaxed(speciesId: number): boolean {
+    // Get this species ID's starter data
+    const starterData = this.scene.gameData.starterData[speciesId];
+
+    return starterData.valueReduction >= 2;
```

Comment:
Magic numbers 👎 
```suggestion
    return starterData.valueReduction >= VALUE_REDUCTION_MAX;
```
(variable has to be defined somewhere central. See if there is other code occurrences for the exact number

Reply:
it is not defined anywhere 💀

Reply:
I know. Thats part of your task